src/components/expanding-link/expanding-link.test.tsx

```diff
@@ -5,38 +5,41 @@ import theme from '@mdb/flora/theme';
 
 import ExpandingLink from '.';
 
-test('renders expanding link with no hover override', () => {
+test('renders expanding link with no hover override', async () => {
     render(<ExpandingLink text="Atlas" href={'/products/atlas/'} />);
 
+    const user = userEvent.setup();
+
     const link = screen.getByRole('link');
     expect(link).toHaveAttribute('href', '/developer/products/atlas/');
     expect(link.getElementsByTagName('span')[0]).toHaveTextContent('Atlas');
 
     expect(link).toHaveStyle({
         right: 0,
     });
-    userEvent.hover(link);
+    await user.hover(link);
     expect(link).toHaveStyle({
         right: `calc(${theme.sizes.inc60} - ${theme.sizes.inc70})`,
     });
 });
 
-test('renders expanding link with hover override', () => {
+test('renders expanding link with hover override', async () => {
     render(
         <ExpandingLink
             text="Atlas"
             href={'/products/atlas/'}
             hoverStyleOverrides={{ top: '10px' }}
         />
     );
+    const user = userEvent.setup();
 
     const link = screen.getByRole('link');
 
     expect(link).toHaveStyle({
         right: 0,
     });
 
-    userEvent.hover(link);
+    await user.hover(link);
```

Comment:
would using `act()` here fit your use case instead of `async/await` ? https://reactjs.org/docs/test-utils.html#act https://flyingsquirrel.medium.com/when-should-i-use-act-in-react-testing-library-d7dd22a3340e